tests/integration_tests/Test/PM/Integration/_data/lock-test/expected-files/lock-mod-h-v3.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "lock-mod-h",
```

Comment:
Nit: naming of these lock files should be more descriptive of their features so its clear what the use is for testing